navigation/screens/facility-form.tsx

```diff
@@ -179,6 +257,10 @@ const styles = StyleSheet.create({
     backgroundColor: "#EEF8F7",
     height: "100%",
   },
+  scrollView: {
```

Comment:
There is a lot of scroll left at the bottom of the page with no content. Can this be fixed?